src/content/docs/integrations/host-integrations/host-integrations-list/redis-monitoring-integration.mdx

```diff
@@ -185,13 +185,16 @@ Config commands include:
     * `keys_limit`: max number of the keys to retrieve their lengths. Default value: `30`. For more information, see [About this integration](#about) and [Keyspace metrics configuration](#keyspace-config).
     * `password`: password to use when connecting to the Redis server. Use only if your Redis server is password-protected. Default value: None.
     * `config_inventory`: set it to `false` to avoid invoking the `CONFIG GET` Redis command. This option is useful in managed environments without permissions to execute the `CONFIG` command (for example, AWS ElastiCache). Default value: `true`
+    * `renamed_commands`: tell the integration to use renamed redis-server commands over defaults. This option is useful in environments where certain redis server commands have been renamed. Example: `renamed_commands: '{"CONFIG": "XYZ"}'`. Default value: `None`
```

Comment:
Saying

> Default value: `None`

might lead to confusion, making users think that the default value is the literal string "None".

What about:

```suggestion
    * `renamed_commands`: tell the integration to use renamed redis-server commands over defaults. This option is useful in environments where certain redis server commands have been renamed. Example: `renamed_commands: '{"CONFIG": "XYZ"}'`. Default value: None.
```